### PR TITLE
fix: minor wordsmith updates to get_started section

### PR DESCRIPTION
**Pull request by jbalsas**
Just some minor corrections to the `Get Started` section. Follow up for adding real info to [How to Use Clay](https://next.clayui.com/docs/get-started/how-to-use-clay.html) created at https://github.com/liferay/clay/issues/2474

### CODE REVIEW

File: next.clayui.com/content/docs/get-started/what-is-clay.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ order: 1
 
 Clay is an extensive Component Library, where you'll find useful components—with extensive code examples—for your web application.
 
-It is composed by 3 different layers, each one of these is really easy to understand and fit perfectly for every need, allowing the developer to choose what to use without limitations
+It is composed by 3 different layers, each one easy to understand and designed to fit different needs, allowing the developer to choose what to use without limitations
```
Comment (wincent) [inline next.clayui.com/content/docs/get-started/what-is-clay.md]: "composed of", not "composed by"

File: next.clayui.com/content/docs/get-started/components-status.md [hunk 2]
```diff
@@ -1,66 +0,0 @@
----
```
Comment (matuzalemsteles) [inline next.clayui.com/content/docs/get-started/components-status.md]: @jbalsas just out of curiosity, why are we removing this?

Comment (jbalsas) [reply next.clayui.com/content/docs/get-started/components-status.md]: this is duplicated and already live at https://github.com/liferay/clay/blob/master/next.clayui.com/content/docs/get-started/how-to-read-this-documentation.md, isn’t it?

Comment (matuzalemsteles) [reply next.clayui.com/content/docs/get-started/components-status.md]: Oh yeah, I was assuming we had just renamed the file name 😅. Thanks @jbalsas

File: next.clayui.com/content/docs/get-started/what-is-clay.md (hunk 1 above)
Comment (jbalsas) [reply next.clayui.com/content/docs/get-started/what-is-clay.md]: pushed!

**Pull request merged**